courses/tests/test_api.py

```diff
@@ -49,3 +57,47 @@ def test_bad_json(self):
 
         resp = self.client.get(modules_url)
         self.assertEqual(resp.status_code, 200, resp.content)
+
+    def test_create_course_accepts_instructors(self):
+        """
+        Validate course create endpoint accepts instructors
+        """
+        resp = self.client.post(reverse('course-list'), {
+            "title": "title1",
+            "author_name": "author1",
+            "overview": "overview1",
+            "description": "description1",
+            "video_url": "http://youtube.com/1",
+            "edx_instance": "http://edx.org/",
+            "instructors": [
+                "861e87a0803e436b989cb62d5e672c5f",
+                "961e87a0803e436b989cb62d5e672c5f"
+            ]
+        })
+        self.assertEqual(resp.status_code, 201)
+        self.assertTrue(EdxAuthor.objects.count(), 2)
```

Comment:
I think you meant to do `assertEqual` here. The second parameter of `assertTrue` is a error message